model-events/src/main/java/de/focus_shift/urlaubsverwaltung/extension/api/application/ApplicationCancelledEventDTO.java

```diff
@@ -0,0 +1,41 @@
+package de.focus_shift.urlaubsverwaltung.extension.api.application;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.Set;
+import java.util.UUID;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+public class ApplicationCancelledEventDTO {
+  @NonNull private UUID id;
+
+  @NonNull private Instant createdAt;
+
+  @NonNull private String tenantId;
+
+  @NonNull private ApplicationPersonDTO person;
```

Comment:
What person is ment here? Maybe applicant?

Reply:
the same, as in the business domain